### PR TITLE
app-crypt/swtpm: Don't install tests when USE="test" , remove old

### PR DESCRIPTION
**Pull request by salahcoronya**
swtpm installs tests to /usr/libexec/installed-tests/swtpm when USE="test" is specified, so delete that directory before installing .

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-crypt/swtpm/swtpm-0.10.0.ebuild [hunk 1]
```diff
@@ -67,6 +67,7 @@ src_configure() {
 
 src_install() {
 	default
+	use test && rm -fr "${D}/usr/libexec/installed-tests/swtpm"
```
Comment (thesamesam) [inline app-crypt/swtpm/swtpm-0.10.0.ebuild:70]: This should be ${ED} but we shouldn't install tests based on USE=test anyway, we should do that based on USE=test-install.

File: app-crypt/swtpm/swtpm-0.10.0.ebuild [hunk 2]
```diff
@@ -67,6 +67,7 @@ src_configure() {
 
 src_install() {
 	default
+	use test && rm -fr "${ED}/usr/libexec/installed-tests/swtpm"
```
Comment (thesamesam) [inline app-crypt/swtpm/swtpm-0.10.0.ebuild:70]: This still needs addressing (USE=test -> USE=test-install), or just always delete it.

Comment (salahcoronya) [reply app-crypt/swtpm/swtpm-0.10.0.ebuild:70]: The directory and its contents are not present if without the "test" use flag. The tests were never intended to be installed into the system.

Comment (thesamesam) [reply app-crypt/swtpm/swtpm-0.10.0.ebuild:70]: OK. Please note that in the commit message.

**Pull request closed without merging**